bin/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  extends: '../.eslintrc.js',
```

Comment:
this should be done with "overrides" in the root, rather than having more than one eslint config file.

Reply:
Done.